.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Primer React Components bug report
```

Comment:
Is it necessary to include "Primer React components" when they're already in the Primer react components repo?
```suggestion
name: Bug report
```